### ISSUE TITLE
DynamicPage: Excessive white space in the header

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents-react#1721]
**Description**
When using only title and header attributes there appears to be extra vertical white space in the header.

**Isolated Example**
```
    <DynamicPage
      data-testid="detailsPage"
      className={styles.header}
      title={
        <DynamicPageTitle
          navigationActions={[
            isFullScreen ? (
              <Button
                key={"exitFullScreen"}
                data-testid="exitFullScreenIcon"
                icon="exit-full-screen"
                design={ButtonDesign.Transparent}
                onClick={onExitFullScreenClick}
                tooltip={i18n.getText("ONBOARDING_DETAILS_EXIT_EXPAND_SCREEN")}
                aria-label={i18n.getText("ONBOARDING_DETAILS_EXIT_EXPAND_SCREEN")}
              />
            ) : (
              <Button
                key={"fullScreen"}
                data-testid="fullScreenIcon"
                icon="full-screen"
                design={ButtonDesign.Transparent}
                onClick={onFullScreenClick}
                tooltip={i18n.getText("ONBOARDING_DETAILS_EXPAND_SCREEN")}
                arial-label={i18n.getText("ONBOARDING_DETAILS_EXPAND_SCREEN")}
              />
            ),
            <Button
              key={"decline"}
              data-testid={"closeDetailsColumn"}
              icon="decline"
              design={ButtonDesign.Transparent}
              onClick={onCloseClick}
              tooltip={i18n.getText("ONBOARDING_DETAILS_CLOSE_DETAILS_SCREEN")}
              aria-label={i18n.getText("ONBOARDING_DETAILS_CLOSE_DETAILS_SCREEN")}
            />
          ]}
        ></DynamicPageTitle>
      }
      header={
        <DynamicPageHeader>
          <FlexBox direction={FlexBoxDirection.Column}>
            <Title level={TitleLevel.H1}>{title}</Title>
            <Label className={styles.headerLabel}>{subTitle}</Label>
          </FlexBox>
        </DynamicPageHeader>
      }
    >
      {children}
    </DynamicPage>
```

**Expected behavior**
If attributes like breadcrumbs etc are not used, they should not consume vertical space.

**Screenshots**
<img width="1920" alt="Screen Shot 2021-06-10 at 3 48 42 PM" src="https://user-images.githubusercontent.com/65705637/121607793-86ca8a80-ca05-11eb-89d8-cde5cd77aec9.png">

Vertical space both above and below screenshots should not be there.

<img width="1919" alt="Screen Shot 2021-06-10 at 3 47 40 PM" src="https://user-images.githubusercontent.com/65705637/121607843-9fd33b80-ca05-11eb-894f-fdf378c17aa4.png">

Also it would be nice if there was a way to set the top/bottom padding of the title bar(below):

<img width="1912" alt="Screen Shot 2021-06-10 at 4 12 33 PM" src="https://user-images.githubusercontent.com/65705637/121608399-b6c65d80-ca06-11eb-9e71-f776fcea01b8.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: ~0.31.9
`@ui5/webcomponents-react` version: ~0.131.7
Operating System: MacOS
Browser: Chrome

**Comment by Lukas742** [comment]
Hi @gurkirpalgill 

thanks for reporting. 
I created a PR that will fix the issue with elements taking up space even though they are not used.

> Also it would be nice if there was a way to set the top/bottom padding of the title bar(below):
> 
> <img alt="Screen Shot 2021-06-10 at 4 12 33 PM" width="1912" src="https://user-images.githubusercontent.com/65705637/121608399-b6c65d80-ca06-11eb-9e71-f776fcea01b8.png">

You can use the `style` prop of the `DynamicPageTitle` component for this: [codeSandbox](https://codesandbox.io/s/gracious-bouman-r8qv6?file=/src/App.js)

**Comment by MarcusNotheis** [comment]
This issue has been fixed in `v0.16.4` and has been downported to `0.131.9` as well.
